Options/WeakAurasOptions.lua

```diff
@@ -7148,14 +7162,48 @@ function WeakAuras.CreateFrame()
     frame.window = "default";
   end
 
+  local function extractTexts(input, ids)
+    local texts = {};
+
+    local currentPos, id, startIdLine, startId, endId, endIdLine;
+    while (true) do
+      startIdLine, startId = string.find(input, L["-- The following part is the\""], currentPos);
```

Comment:
@InfusOnWoW 

`-- Do not remove this comment, it is part of this trigger: XXX`
Ok maybe this then?